src/ci/resources/papi_v2beta_application.conf

```diff
@@ -38,6 +38,27 @@ services {
           max-interval = 15 seconds,
           multiplier = 1.1
         }
+
+        # Metadata deletion configuration.
+        metadata-deletion {
```

Comment:
ooc why is all of this copy/pasted into two files for alpha and beta rather than in a common include?

Reply:
Unfortunately I lost my "No Time To D.R.Y." file when I moved laptops, so I have no excuse not to do this any more